### ISSUE TITLE
prettier cannot deal with class instantiation from a config

### ISSUE THREAD

**Issue by snake-py** [prettier/plugin-php#2082]
Prettier cannot format the following code:

```php
        $client = new (config('wd.authentication.models.client')(['name' => $data['name']]));
```
I am getting:

```php
[error] src/Http/Controllers/RegisterController.php: SyntaxError: Parse Error : syntax error, unexpected '(' on line 48
[error]   46 |     protected function create(array $data): BaseUser
[error]   47 |     {
[error] > 48 |         $client = new (config('wd.authentication.models.client')(['name' => $data['name']]));
[error]      |                      ^
[error]   49 |         $client->save();
[error]   50 |
[error]   51 |         $user = $this->getUserObject()::create($this->getNewUserData($data));
```

WorkAround:

```php
        $ClientModel = config('wd.authentication.models.client');
        $client = new $ClientModel(['name' => $data['name']]);
```

**Comment by cseufert** [comment]
Looks to more of an upstream parser issue.

**Comment by czosel** [comment]
Fixed in `v0.19.2` :tada:
